src/main/java/xyz/nucleoid/plasmid/entity/FloatingText.java

```diff
@@ -43,7 +43,7 @@ public void setText(Text line) {
         this.setText(new Text[] { line });
     }
 
-    // TODO: 0.5- api needs to better inform caller whether spawning failed
+    // TODO: 0.5- migrate to patbox's floating text api
```

Comment:
The question is, do we want to just rebuild the implementation of this on top of that Holograms API or do we just deprecate and encourage the use of the Holograms API instead?

Reply:
I think we can rather just include the holograms api- since this is a breaking version anyway, we may as well remove deprecated stuff directly.